src/Text/Pandoc/App/CommandLineOptions.hs

```diff
@@ -727,9 +728,11 @@ options =
 
      , Option "" ["csl"]
                  (ReqArg
-                  (\arg opt ->
-                     return opt{ optMetadata =
-                                   addMeta "csl" (normalizePath arg) $
+                  (\arg opt -> do
+                    case lookupMeta (T.pack "csl") $ optMetadata opt of
+                      Just meta -> E.throwIO $ PandocOptionError $
+                         "Only one CSL file can be specified." <> maybe "" (" Ignoring " <>) (metaValueToText meta)
+                      Nothing -> return opt{ optMetadata = addMeta "csl" (normalizePath arg) $
```

Comment:
I think that if this becomes an error instead of a warning, then the "Ignoring ..." message is just going to be misleading.  (After all, the program stopped; it's ignoring both --csl arguments.)

So, I'd say, either we go with a warning and say what we're ignoring, or go with an error and just say "Only one CSL file can be specified." The user will figure it out from there.

Reply:
Thanks, good point. I went with the error and fixed the message.

Reply:
ok, thanks.